### PR TITLE
Add blank VSchema for new keyspaces

### PR DESCRIPTION
**Pull request by dkhenry**
This change adds a blank VSchema for a newly created keyspace. As
keyspaces can't be used until they have a vschema this allows an
unsharded keyspace to be used without requireing an explicit vschema
to be loaded for it

Signed-off-by: Dan Kozlowski <koz@planetscale.com>

### CODE REVIEW

**Comment by dkhenry** [comment]
I have moved the logic to vtctl, and it can be turned off by setting `-allow_empty_vchema`. Also It looks like DeleteKeyspace already removes the VSchema by default. The delete does rely on the behaviour that giving an empty VSchema to `SaveVSchema` restults in the node being deleted.

Also in terms of the tests. I am thinking that sounds like it should be an e2e test so I will put something together there.

File: go/vt/topo/srv_vschema.go [hunk 1]
```diff
@@ -134,3 +138,78 @@ func (ts *Server) DeleteSrvVSchema(ctx context.Context, cell string) error {
 	nodePath := SrvVSchemaFile
 	return conn.Delete(ctx, nodePath, nil)
 }
+
+// RebuildVSchema rebuilds the SrvVSchema for the provided cell list
```
Comment (demmer) [inline go/vt/topo/srv_vschema.go]: With the change to move the logic to rebuild the vschema into vtctl, do we still need to move RebuildVSchema from topotools into the server itself?

File: go/vt/vtctl/vtctl.go [hunk 2]
```diff
@@ -1576,6 +1578,26 @@ func commandCreateKeyspace(ctx context.Context, wr *wrangler.Wrangler, subFlags
 		wr.Logger().Infof("keyspace %v already exists (ignoring error with -force)", keyspace)
 		err = nil
 	}
+
+	vschema, err := wr.TopoServer().GetVSchema(ctx, keyspace)
+	if *allowEmptyVSchema == false && (vschema == nil || err != nil) {
```
Comment (demmer) [inline go/vt/vtctl/vtctl.go]: The err != nil check here seems problematic -- if for some reason there is a vschema but there was an error fetching it, then this path would stomp on the existing one. Probably not a huge deal, but I'd rather us explicitly check for the err indication that the vschema doesn't exist and not blanket fail on all errors.

File: go/vt/topo/srv_vschema.go (hunk 1 above)
Comment (dkhenry) [reply go/vt/topo/srv_vschema.go]: I would prefer to keep this as it was a simple refactor and it makes more sense to have this as a member function of topo and not part of topotools. I think there may have been concerns with import loops, but by moving the test here to topotest I was able to resolve the import loop

**Comment by dkhenry** [comment]
@demmer @enisoc @sougou This is ready for review. I have added a e2e test for unsharded keyspaces and testing the basics of DDL's for them.

**Pull request merged**